arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -180,3 +183,25 @@ extension UIColor {
     // The translucent background color for status labels.
     class var statusLabelBackgroundColor: UIColor { return UIColor(named: "statusLabelBackgroundColor")! }
 }
+
+extension AppDelegate {
+    /// License the app with ArcGIS Runtime deployment license keys.
+    ///
+    /// - Note: An invalid key does not throw an exception, but simply fails to license the app,
+    ///   falling back to Developer Mode (which will display a watermark on the map view).
+    static func licenseApplication() {
+        do {
+            // Don't set extension key if you don't use utility network samples.
+            // try AGSArcGISRuntimeEnvironment.setLicenseKey(.licenseKey)
+            
+            // Set both keys for accessing all samples.
+            try AGSArcGISRuntimeEnvironment.setLicenseKey(.licenseKey, extensions: [.extensionLicenseKey])
+            
+            // Set API keys for new metered basemap styles.
+            // AGSArcGISRuntimeEnvironment.apiKey = .apiKey
+        } catch {
+            print("[Error: AGSArcGISRuntimeEnvironment] Error licensing app: \(error.localizedDescription)")
+        }
+        print("[ArcGIS Runtime License] \(AGSArcGISRuntimeEnvironment.license())")
```

Comment:
Please remove this print statement.

Reply:
Hi Phil, what type of error handling do you think we need to do here? 

- In `Data Collection` app by Eli it prints out licensing information
- In our original iOS Release Instructions wiki, it also prints out the info/error

However, since it is not our goal to help users to understand the details of licensing process, how about simply put a `fatalError` in the catch block?

Reply:
I was referring specifically to the print statement on line `205` (the one that prints the license). The one inside the catch statement is fine, although I would remove the `[Error: AGSArcGISRuntimeEnvironment] ` portion.